sdcm/utils/tablets/common.py

```diff
@@ -23,9 +23,10 @@ def __str__(self):
         return '{' + ', '.join(items) + '}'
 
 
-def wait_for_tablets_balanced(node):
+def wait_no_ongoing_topology_operations(node):
     """
-    Waiting for tablets to be balanced using REST API.
+    Waiting for having no ongoing tablets topology operations using REST API.
```

Comment:
worth to add this note you mentioned, that waits for current migration to complete (still another one may start in a second)

Reply:
done